server/cloudformation/test.go

```diff
@@ -0,0 +1,33 @@
+package cloudformation
```

Comment:
May wanna rename this `<prefix>_test.go` otherwise the imports will get included in the binary and the types would be available at runtime.